csharp/ql/src/semmle/code/csharp/dataflow/FlowSummary.qll

```diff
@@ -70,14 +70,14 @@ module SummaryComponentStack {
   /** Gets a singleton stack representing a qualifier. */
   SummaryComponentStack qualifier() { result = singleton(SummaryComponent::qualifier()) }
 
-  /** Gets a stack representing an element of `of`. */
-  SummaryComponentStack elementOf(SummaryComponentStack of) {
-    result = push(SummaryComponent::element(), of)
+  /** Gets a stack representing an element of `container`. */
+  SummaryComponentStack elementOf(SummaryComponentStack container) {
+    result = push(SummaryComponent::element(), container)
   }
 
-  /** Gets a stack representing a propery `p` of `of`. */
-  SummaryComponentStack propertyOf(Property p, SummaryComponentStack of) {
-    result = push(SummaryComponent::property(p), of)
+  /** Gets a stack representing a propery `p` of `object`. */
```

Comment:
You can apply the same change a couple of lines below for fields.

Reply:
Ah, yes. Done.